dvr_scan/scanner.py

```diff
@@ -308,19 +320,23 @@ def scan_motion(self):
                     num_frames_read += 1
                     if progress_bar:
                         progress_bar.update(1)
+                    self.frames_read = num_frames_read
```

Comment:
I added this property to the class so the GUI can access this to calculate the progress.